src/AlgebraicGeometry/Schemes/ProjectiveVariety/Objects/Methods.jl

```diff
@@ -19,7 +17,7 @@ function Base.show(io::IO, X::AbsProjectiveVariety{<:Field,<:MPolyQuoRing})
   elseif get_attribute(X, :is_empty, false)
     print(io, "Empty projective variety")
   else
-    print(io, "V(")
+    print(io, LowercaseOff(), "V(")
```

Comment:
This ensures the `V` always is in uppercase, so technically this is a bug fix, albeit a very minor one

Reply:
Good to have this function, I was not aware of this one, thanks !